ui.py

```diff
@@ -27,8 +27,10 @@ def show_books(books):
     # TODO print a blank line before and after the book list or no books message
```

Comment:
We ought to remove the associated 'TODO' once it's todone

Reply:
TODO message has been removed and empty lines have been modified